README.md

```diff
@@ -5,11 +5,1119 @@
 
 [![CircleCI](https://circleci.com/gh/prisma-labs/nexus-prisma.svg?style=svg)](https://circleci.com/gh/prisma-labs/nexus-prisma)
 
-`nexus-prisma` is a plugin for bridging [Prisma](https://www.prisma.io) and [Nexus](https://nexus.js.org). It extends the Nexus DSL `t` with `.model` and `.crud` making it easy to expose Prisma models and operations against them in your GraphQL API. The resolvers for these operations (pagination, filtering, ordering, and more), are dynamically created for you removing the need for traditional ORMs/query builders like TypeORM, Sequelize, or knex. And when you do need to drop down into custom resolvers a [Photon](https://photonjs.prisma.io) instance on `ctx` will be ready to serve you, the same great tool `nexus-prisma` itself bulids upon.
+`nexus-prisma` is a plugin for bridging [Prisma](https://www.prisma.io) and [Nexus](https://nexus.js.org). It extends the Nexus DSL `t` with `.model` and `.crud` making it easy to expose Prisma models and operations against them in your GraphQL API. The resolvers for these operations (pagination, filtering, ordering, and more), are dynamically created for you removing the need for traditional ORMs/query builders like TypeORM, Sequelize, or Knex. And when you do need to drop down into custom resolvers a [Photon](https://photonjs.prisma.io) instance on `ctx` will be ready to serve you, the same great tool `nexus-prisma` itself bulids upon.
 
-## Examples
+**Table of Contents**
```

Comment:
I have to make doctoc + vsc happy. doctoc ignores titles above the toc, and vsc permits a comment to ignore titles 👍 

Are you ok with an H3? I feel H1 competes with the title and H2 having the horizontal bar adds too move "bar noise" at the top. Also a bit of context: I was thinking the opening text should be pretty minimal and not actually constitute a section in its own right. Just a bit of feeler text. Having it just flow casually into the contents rather than it being these big bold distinct sections.

![image](https://user-images.githubusercontent.com/284476/66652691-48634380-ec04-11e9-90e3-01c11ed27d50.png)

![image](https://user-images.githubusercontent.com/284476/66651852-16e97880-ec02-11e9-944b-033c1309ce40.png)

![image](https://user-images.githubusercontent.com/284476/66651867-1fda4a00-ec02-11e9-922d-c9d14341fae6.png)